src/BigwigDataFetcher.js

```diff
@@ -103,9 +103,22 @@ const BBIDataFetcher = function BBIDataFetcher(HGC, ...args) {
 
     loadBBI(dataConfig) {
       if (dataConfig.url) {
-        this.bwFile = new BigWig({
-          filehandle: new RemoteFile(dataConfig.url),
-        });
+        // look for Basic credentials in the URL; if present, extract
+        // them and use them to create an authenticated RemoteFile object
+        let url = new URL(dataConfig.url);
+        const username = url.username;
```

Comment:
Could you update the README to describe how this should be used and under what circumstances?

Reply:
README has been updated with a short description of Basic authentication and how to change the URL to add credentials in the view configuration.